packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -451,7 +458,7 @@ const Dropdown = ({
                         className={dropdownDisplayClassName()}
                         data-testid='dt_dropdown_display'
                         tabIndex={isSingleOption() ? -1 : 0}
-                        onClick={handleVisibility}
```

Comment:
<img width="318" alt="image" src="https://github.com/binary-com/deriv-app/assets/56330681/3b4295b6-d132-4499-8259-7e2802e1caa9">
When we open the dropdown on mobile it doesn't scroll to the selected option. Can we fix this?

Reply:
Yes, we can. Actually I'm not sure, how it should be because in design it's not seems that it should be scrolled.
<img width="382" alt="Screenshot 2023-12-26 at 11 49 03" src="https://github.com/binary-com/deriv-app/assets/121025168/9a3c6929-53e9-4a35-88bd-7ad2ef79f2c3">

Reply:
we should automatically scroll to selected option.

Reply:
Done